stdlib/tkinter/__init__.pyi

```diff
@@ -728,18 +728,29 @@ class _ExceptionReportingCallback(Protocol):
 
 class Tk(Misc, Wm):
     master: None
-    def __init__(
-        # Make sure to keep in sync with other functions that use the same
-        # args.
-        # use `git grep screenName` to find them
-        self,
-        screenName: str | None = None,
-        baseName: str | None = None,
-        className: str = "Tk",
-        useTk: bool = ...,
-        sync: bool = ...,
-        use: str | None = None,
-    ) -> None: ...
+    # Make sure to keep __init__ in sync with other functions that use the same args.
+    # Use `git grep screenName` to find them
+    if sys.version_info >= (3, 9):
+        def __init__(
+            self,
+            screenName: str | None = None,
+            baseName: str | None = None,
+            className: str = "Tk",
+            useTk: bool = True,
+            sync: bool = False,
+            use: str | None = None,
+        ) -> None: ...
+    else:
+        def __init__(
+            self,
+            screenName: str | None = None,
+            baseName: str | None = None,
+            className: str = "Tk",
+            useTk: bool = ...,
```

Comment:
Can't we just allowlist this? It would be useful to see `True` and `False` in IDEs, even if you don't happen to be using a new Python version.

Reply:
Sure!

Reply:
No allowlist additions needed for now, but we'll have to add some when mypy 1.0 comes out and stubtest starts verifying default values.